datanode/bootstrap/bootstrap_details.go

```diff
@@ -0,0 +1,84 @@
+package bootstrap
+
+import (
+	"encoding/json"
+	"sync"
+)
+
+type status int
+
+const (
+	notApplicable status = iota
+	copyNeeded
+	copyFinished
+)
+
+type bootstrapDetails struct {
```

Comment:
will update debug ui to output bootstrap detail?

Reply:
yeah. will add a new ui tab for bootstrap